src/context/CartContext.js

```diff
@@ -0,0 +1,41 @@
+import axios from 'axios';
+import {createContext} from 'react'
+import {useContext,useState,useEffect} from 'react'
```

Comment:
Both import from react only. Use VS code intelli or ES6/7 extesion for auth importing.

Reply:
will work on this vijay.thank you